server/src/test/java/io/crate/planner/operators/JoinTest.java

```diff
@@ -677,14 +677,14 @@ CROSS JOIN UNNEST([1]) sensors(sensor_id)
         assertThat(logicalPlan).isEqualTo(
             """
             Eval[time, sensor_id, battery_level]
-              └ NestedLoopJoin[LEFT | ((time = time) AND (sensor_id = sensor_id))]
-                ├ NestedLoopJoin[CROSS]
-                │  ├ Rename[time] AS time_series
-                │  │  └ TableFunction[generate_series | [generate_series] | true]
-                │  └ Rename[sensor_id] AS sensors
-                │    └ TableFunction[unnest | [unnest] | true]
-                └ Rename[battery_level, time, sensor_id] AS readings
-                  └ Collect[doc.sensor_readings | [battery_level, time, sensor_id] | true]
+              └ NestedLoopJoin[RIGHT | (time = time)]
```

Comment:
I've tested this against postgres with various values inserted in the `sensor_readings` table and we produce the same results as postgres.